README.rst

```diff
@@ -20,4 +20,98 @@ Python implementation of PVL (Parameter Value Language)
 Features
 --------
 
-* TODO
\ No newline at end of file
+* TODO
+
+How to use Module
+--------------------
+
+.. contents:: `Table of Contents`
+	:local:
+
+pvl.load
++++++++++
+
+General:: 
+
+This module parses an isis label from a stream and returns a dictionary 
+containing information from the label. The stream must be a string of the image 
+file name with the path to the file included.
+
+ >>> from pvl import load
```

Comment:
Yeah, since the module and function name are short, I think that makes sense and is the most explicit way to do things.